src/mongo/mongo-eventstore-adapter.ts

```diff
@@ -0,0 +1,23 @@
+import { IAdapterStore } from '@juicycleff/nestjs-event-store';
+import { connect } from './database';
+import { StoreStateModel } from './store-state.model';
+
+export function MongoStore(key: string): IAdapterStore {
+  connect();
+
+  return {
+    storeKey: key,
+    write: async (streamName: string, checkpoint: number) => {
+      const updateCheckpoint = await StoreStateModel.updateOneOrCreate({
```

Comment:
could've written to file on disk here as well of course. 
game.txt with just the number in it. Overwrite every time.

Reply:
on every event now a write to the MongoDB. performance issue?